ephemeral/startup/action.yml

```diff
@@ -53,11 +58,10 @@ runs:
         # TODO: make preview name configurable!
         previewName=preview-$prId
 
-        response=$(curl -X POST -d "{\"auto_load_pod\": \"${AUTO_LOAD_POD:-${{ inputs.auto-load-pod }}}\"}" \
+        response=$(curl -X POST -d "{\"instance_name\": \"${previewName}\", \"env_vars\": {\"AUTO_LOAD_POD\": \"${AUTO_LOAD_POD:-${{ inputs.auto-load-pod }}}\", \"EXTENSION_AUTO_INSTALL\": \"${EXTENSION_AUTO_INSTALL:-${{ inputs.extension-auto-install }}}\"}}"\
```

Comment:
nit: This is pretty dense syntax, maybe we could pull the conditional assignment for `AUTO_LOAD_POD` / `EXTENSION_AUTO_INSTALL` into temporary variables?

Also, just to confirm - do we want the `inputs.*` values to be the fallback (if the env variables are not defined), or rather the other way around, i..e, have them overwrite the env. variables, if configured? 🤔